<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AjoutEmploye</class>
 <widget class="QDialog" name="AjoutEmploye">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>679</width>
    <height>657</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hotel</string>
  </property>
  <property name="windowIcon">
   <iconset resource="image.qrc">
    <normaloff>:/projetimages/projetimages/e.png</normaloff>:/projetimages/projetimages/e.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">
QDateEdit {
    border: 1px solid gray;
    border-radius: 3px;
    padding: 1px 18px 1px 3px;
    min-width: 6em;
}
QLineEdit
{

    border: 1px solid gray;
    border-radius: 4px;
    padding: 1px 18px 1px 3px;
    min-width: 6em;
}
QDialog#AjoutEmploye
{
	background-image: url(:/projetimages/projetimages/a.jpg);

}
*{
font-family:centry gothlic ;
font-size:15px;
}
QWidget#widget
{

	background:rgba(0,0,0,0.7);
border-radius:15px;
}


QPushButton
 {
     background-color: red;

     border-radius: 10px;

 }

#label{
color:white;
background:transparent;

}

#label_2
{
color:white;
background:transparent;

}


#label_3{
color:white;
background:transparent;

}

#label_4{
color:white;
background:transparent;

}

#label_5{
color:white;
background:transparent;

}

#label_6{
color:white;
background:transparent;

}

#label_7{
color:white;
background:transparent;

}

#label_8{
color:white;
background:transparent;

}



QPushButton
{
color:white;

border-raduis:15px;
}
QPushButton:hover
{

	color:white;
border-radius:15px;
background:rgb(255, 85, 127);
}
</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>39</x>
     <y>29</y>
     <width>591</width>
     <height>611</height>
    </rect>
   </property>
   <widget class="QLineEdit" name="mdp2empaj_txt">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>460</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>410</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Mot de passe :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="adresseempajout_txt">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>290</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="loginempaj_txt">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>350</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nom :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>180</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>CIN :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="cinempaj_txt">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>170</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="mdp1emaj_txt">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>410</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="nomempaj_ttxt">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>60</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>120</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Pr√©nom :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>240</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Date de naissance :</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="datenaissanceemaj_txt">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>230</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="prenomempaji_txt">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>110</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>300</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Adresse :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>350</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Login :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ajouteremp_btn">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>540</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Enregistrer</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>460</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Confirmer le mot de passe :</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
